### PR TITLE
[FIX] SyntaxError: Unexpected token '.' in Node.js v12

### PR DESCRIPTION
**Pull request by matz3**
Regression caused by https://github.com/SAP/ui5-cli/pull/533

Fixes: https://github.com/SAP/ui5-tooling/issues/663

### CODE REVIEW

**Review by RandomByte — DISMISSED** [review]

**Review by RandomByte — DISMISSED** [review]

**Review by flovogt — APPROVED** [review]

**Review by flovogt — CHANGES_REQUESTED** [review]
Looks like the Yargs Error handling does not longer work as expected

**Comment by matz3** [comment]
I'll have another look into the tests

File: bin/ui5.js [hunk 1]
```diff
@@ -26,16 +26,31 @@ if (
...
+			console.log("====================== UNSTABLE NODE.JS VERSION =====================");
+			console.log("You are using an unstable version of Node.js");
+			console.log("Detected Node.js version: " + nodeVersion);
+			console.log("");
+			console.log("=> Please note that an unstable version might cause unexpected");
+			console.log("   behavior. For productive use please consider using a stable");
+			console.log("   version of Node.js! For the release policy of Node.js, see");
+			console.log("   https://nodejs.org/en/about/releases");
+			console.log("=====================================================================");
+		} else if (semver.satisfies(nodeVersion, "< 14.16 || ^15 || ^17")) {
```
Comment (flovogt) [inline bin/ui5.js:40]: `< 14.16` means also node version `10` and `12`, correct?

Comment (RandomByte) [reply bin/ui5.js:40]: Yes:
```
> s.satisfies("10.0.0", "< 14.16")
true
```

**Review by RandomByte — APPROVED** [review]

**Review by flovogt — APPROVED** [review]

**Pull request merged**